nilearn/decoding/tests/test_decoder.py

```diff
@@ -102,23 +103,22 @@ def test_BaseDecoder_check_estimator_validity():
                             'ridge', 'ridge_classifier',
                             'ridge_regressor', 'svr']
     unsupported_estimators = ['ridgo', 'svb']
-    expected_warnings = ['Use a custom estimator at your own risk '
-                         'of the process not working as intended.']
+    expected_warnings = 'Use a custom estimator at your own risk ' \
+                        'of the process not working as intended.'
```

Comment:
Surround the warning string in parentheses instead of using a line continuation.
When there is only one warning, call it `expected_warning`.